src/components/layertree/LayerTree.js

```diff
@@ -242,7 +271,7 @@ class LayerTree extends PureComponent {
   /**
    * Apply a mutation to all the children recursively.
    */
-  applyToChildren(tree, item, mutation, itemIgnored) {
+  applyToChildren(tree, item, mutation, isIgnoredFunc, isCheckboxFunc) {
```

Comment:
I understand better now maybe a function named `isTypeIgnoredFunction` makes more sense.

Reply:
I apply this naming in the next MR.

Reply:
I ve done it directly on github.